spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/PubSubAdmin.java

```diff
@@ -235,22 +234,33 @@ public Subscription createSubscription(String subscriptionName, String topicName
 		Assert.hasText(subscriptionName, "No subscription name was specified.");
 		Assert.hasText(topicName, NO_TOPIC_SPECIFIED);
 
-		int finalAckDeadline = this.defaultAckDeadline;
+		Subscription.Builder builder = Subscription.newBuilder()
+				.setName(PubSubSubscriptionUtils.toProjectSubscriptionName(subscriptionName, this.projectId).toString())
+				.setTopic(PubSubTopicUtils.toTopicName(topicName, this.projectId).toString());
+
+		builder.setAckDeadlineSeconds(this.defaultAckDeadline);
```

Comment:
Hmmm, previously we would only set ack deadline if the user specified a custom one. Was that a bug?

Reply:
It's not true that we previously didn't set the ack deadline - see the actual `createSubscription` method call in the previous code - we were always passing the default or the user override.

Reply:
You're right!